website/content/vagrant-cloud/api.mdx

```diff
@@ -132,23 +135,24 @@ curl \
 # Prepare the provider for upload/get an upload URL
 
 response=$(curl \
+    --request GET \
     --header "Authorization: Bearer $VAGRANT_CLOUD_TOKEN" \
- https://app.vagrantup.com/api/v1/box/myuser/test/version/1.2.3/provider/virtualbox/upload)
+    https://app.vagrantup.com/api/v1/box/myuser/test/version/1.2.3/provider/virtualbox/upload)
 
 # Extract the upload URL from the response (requires the jq command)
 
 upload_path=$(echo "$response" | jq .upload_path)
 
 # Perform the upload
 
-curl $upload_path --request PUT --upload-file virtualbox-1.2.3.box
+curl --request PUT $upload_path --upload-file virtualbox-1.2.3.box
```

Comment:
Would be good to wrap this in quotes since it's getting updated (`"${upload_path}"`)

Reply:
Done!